testing/MLDB-1012_nested_function_calls.py

```diff
@@ -79,14 +79,14 @@
 js_rez = rez.json()
 assert js_rez[0]['columns'][0][1] == 1
 
-mldb.put("/v1/functions/recurse", {
-    "type": "sql.expression",
-    "params": {"expression": "recurse({input})[output] as output"}
-})
-
 # MLDB-1251
 mldb.log("MLDB-1251")
 try:
+    mldb.put("/v1/functions/recurse", {
+        "type": "sql.expression",
+        "params": {"expression": "recurse({input})[output] as output"}
+    })
```

Comment:
This tested that the query failed. From the change I guess that it now tests that the put function fails instead. So it doesn't test the same thing.